File: js/sumOfMinimums.js
// Problem# 149
// Create a function which takes a 2D array arr returns the sum of minimum value in each row.
// https://edabit.com/challenge/5RiZh6yJKeGEo4P3z

const sumMinimums = (arr) =>
  arr.map((item) => Math.min(...item)).reduce((a, b) => a + b, 0);

module.exports = sumMinimums;

File: python/sum_of_minimums.py
"""
Sum of Minimums

Create a function that takes a 2D list lst returns the sum of the minimum value in each row.

https://edabit.com/challenge/5NMfCfBNqw3iiDeeR
"""
from typing import List


def sum_minimums(lst: List[List[int]]) -> int:
    return sum(min(lst_item) for lst_item in lst)
